### ISSUE TITLE
模型准确率计算有问题

### ISSUE THREAD

**Issue by shineyjg** [nickliqian/cnn_captcha#16]
修正为即可：
accuracy = tf.reduce_mean(tf.reduce_min(tf.cast(correct_pred, tf.float32), axis=1))
我已经提了pull request，作者有空可以合下代码

**Comment by nickliqian** [comment]
reduce_min是取最小值，在这里就是取0，这样的话准确率一直输出0，所以就不合并代码了~

**Comment by shineyjg** [comment]
原算法计算每一位准确率，比如对一个4位数字验证码，有3位正确，1位错误，原算法准确率75%。改正后以每个样本为单位，一个验证码只有全部4位正确才算1次正确，反之，一个样本只要有错误就算1次错误。如果有1000个样本，共4000位数字。原算法计算4000位数字有几个正确，改正后算法计算1000个样本中有几个正确

**Comment by nickliqian** [comment]
已经合并代码

**Comment by lazy0327** [comment]
这种新增的方式应该会使得前面的训练基本准确率都为0，哪种会比较好呢？

**Comment by wangyeqiang** [comment]
我觉得之前的好点呢- -

**Comment by nickliqian** [comment]
> 我觉得之前的好点呢- -

是不同的思路哈~
